back/app/services/xlsx_export/custom_field_for_report.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module XlsxExport
+  class CustomFieldForReport
```

Comment:
Thank you :-)

If they were one class, the methods would have several `if` statements, which complicates them. The important part is that they share the same API, so that the generation of a column header or a value does not have to make the distinction.